examples/with-cloudinary/next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  images: {
+    domains: ["res.cloudinary.com"],
```

Comment:
This should probably use the `loaderFile` to to talk to cloudinary directly. That way users can specify the options they intend to use.

Reply:
We tried adding the Cloudinary loader and it ended up being slower so we're omitting it.